Add ES-notebook as script

thomasht86 opened the pull request:
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

- [x] Used `filter_path` to get only needed fields from response
- [x] Use `app_es/index-settings.json` for config. 
- [x] Made the notebook (script) self-contained (except from index settings)  
- [x] Use threading to parallellize uploading. 
- [x] Upload chunks directly from decompress stream to avoid unzipping large file. 
- [x] Added dynamic download from S3-url. (Not tested that all configurations are present there yet)

Ignore the 'run_benchmarks_vespa.py' for now, as it is WIP. 

- Removed the notebook now @geirst

geirst reviewed (commented):
I have only reviewed `run_es_benchmarks.py`. Consider removing `run_vespa_benchmarks.py` from this PR until it is complete.

thomasht86 commented on tests/performance/ecommerce_hybrid_search/common/run_es_benchmarks.py:
```diff
@@ -0,0 +1,358 @@
+            "-P", # Post requests (GET is default)
+            "-o", # Output file
+            config["output_file"],
+            "-D", # Use TLS configuration from environment
```
it worked in docker, but better to remove it then.

thomasht86 commented:
Added the script for running vespa benchmarks as script and removed notebook. 
Delaying refactoring between notebooks for now, as it might be a wasted effort, until we know what we are doing with this.

geirst commented on tests/performance/ecommerce_hybrid_search/common/run_vespa_benchmarks.py:
```diff
@@ -0,0 +1,285 @@
+
+# Use vespa CLI (vespa feed --progress 10) to feed data
+set_target_command = "vespa config set target local"
+feed_command = f"vespa feed --progress 10 {feed_file_vespa.replace('.zst', '')}"
```
Ideally we should use `vespa-feed-client` to be on par with the performance test, but we can keep this for now.

geirst reviewed (approved):
👍
